elastic_search_mapping.json

```diff
@@ -8,6 +8,8 @@
         "equity_release": { "type": "boolean" },
         "inheritance_tax_planning": { "type": "boolean" },
         "wills_and_probate": { "type": "boolean" },
+        "other_advice_methods": { "type": "long" },
```

Comment:
I've actually taken them from auto-generated definitions. Will amend the commit.